resources/metadata/helmExecute.yaml

```diff
@@ -360,6 +360,13 @@ spec:
         scope:
           - STEPS
           - PARAMETERS
+      - name: renderValuesTemplate
+        type: bool
+        description: A flag to turn templating value files on or off.
+        default: true
```

Comment:
Does it work like this? Setting the default (only) here in the helmExecute.yaml and it works everywhere?
How is this yaml used? It the tests the defaults seems to be not true. Do we have to set the default also somewhere else?

(A parameter "skipRenderValuesTemplate" would be simpler because after initializing the `bool` from an "unset" value would probably be "false". But I agree the parameter should be `renderValuesTemplate`)

Reply:
The default is also set in the generated files.